### PR TITLE
Adding view mode for enum criterias

### PR DESCRIPTION
**Pull request by cesar82**

### CODE REVIEW

File: src/components/molecules/EditableList/TableVersion/style.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const StyledTable = styled(Table)`
   
   &.table-sm {
     td, th {
-      vertical-align: middle;
+      vertical-align: top;
```
Comment (normandanielmq) [inline src/components/molecules/EditableList/TableVersion/style.js:10]: Are the row numbers in edit-mode still aligned after this change?

File: src/components/pages/ProgramDetailsPage/CriteriaListEditor/index.js [hunk 2]
```diff
@@ -234,24 +279,28 @@ const CriteriaListEditor = (
           visible={usedCriteria.academics}
         />
         <Activity
-          {...{ criteriaType, control, watch, renderMessage }}
+          {...criteriaProps}
+          {...{ control, watch }}
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/CriteriaListEditor/index.js:283]: criteriaProps already includes formMethods

File: src/components/molecules/EditableList/TableVersion/style.js (hunk 1 above)
Comment (cesar82) [reply src/components/molecules/EditableList/TableVersion/style.js:10]: I adjusted the row number column to have vertical-align middle

**Review by fabian202 — APPROVED** [review]

**Review by normandanielmq — APPROVED** [review]

File: src/components/pages/ProgramDetailsPage/EnumCriteria/index.js [hunk 3]
```diff
@@ -152,7 +137,19 @@ const EnumCriteria = ({
...
+      title={criteriaName}
+      {...{
+        visible,
+        ref,
+        editedSection,
+        onEdit,
+        criteriaType,
+        onCancel,
+        loading,
+      }}
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/EnumCriteria/index.js:151]: Criteria component uses formMethods prop

**Pull request merged**